attributes/default.rb

```diff
@@ -24,28 +17,18 @@
 # Will be populated automaticaly when roles use, unless overrided.
 default['stackstorm']['components'] = %w(st2common)
 default['stackstorm']['service_binary'] = {}
+default['stackstorm']['install_stackstorm']['version'] = '1.4.0-8'
```

Comment:
In [Ansible repo](https://github.com/stackstorm/ansible-st2) we follow automatic updates approach, eg avoid pinning like that.

If users want to pin it to numeric version like `1.4.1` or even pin build number, - they can optionally do it.

And yeah, updating other `N` configuration management repositories just to change every major/minor/build number is not convenient.

I believe that installing `st2` latest stable version (or even better - major version like `1.3`, `1.4`, etc if we care about making older `chef-stackstorm` versions to work), - that's probably the best default for users.